pages.fr/common/ansible-pull.md

```diff
@@ -0,0 +1,20 @@
+# ansible-pull
+
+> Récupère les playbook ansible depuis un dépôt VCS et exécute les en local.
+> Plus d'informations : <https://docs.ansible.com/ansible/latest/cli/ansible-pull.html>.
+
+- Récupère le playbook depuis un VCS exécute le fichier par défaut local.yaml du playbook :
```

Comment:
Something is missing. Maybe this version?

```suggestion
- Récupère le playbook depuis un VCS et exécute le fichier par défaut local.yaml du playbook :
```

Reply:
nice catch !